sequential/maps/README.md

```diff
@@ -46,3 +46,46 @@ to_map([]).
 %% #{}
 ```
 [solution](src/solution/list_to_map.erl).
+
+### Records to Maps
+
+Create a record named `person` that has the attributes `name` and `age`. Then write a function `map:records_to_maps(Records)` that converts a list of records (in this case people) into a list of maps with the attributes. For this you should use `lists:map`.
```

Comment:
`map` is not a great choice to name a module, since Erlang already provides a `maps`. I'd consider using something erlings specific.

Reply:
`map` was already used as the module name for the exercise. I would prefer to leave it as is and in a separate issue propose the name change because.

Reply:
don't use map. change it